backend/app/controllers/enrollments_controller.rb

```diff
@@ -224,6 +224,26 @@ def next_enrollments
       root: "enrollments"
   end
 
+  # GET enrollment/1/mark_demandeur_notify_events_as_processed/1
+  def mark_demandeur_notify_events_as_processed
+    @enrollment = policy_scope(Enrollment).where(id: params[:id])
+    # @enrollment = authorize Enrollment.find(params[:id])
+    demandeurs_user_id = @enrollment.demandeurs.pluck(:user_id)
```

Comment:
```suggestion
    demandeur_user_ids = @enrollment.demandeurs.pluck(:user_id)
```

c'est plus conventionnel comme nom de variable quand on manipule des IDs (c'est ce que fait ActiveRecord)

Reply:
Merci :) Je vais changer cela.